src/views/Play/Play.tsx

```diff
@@ -73,6 +73,9 @@ export class Play extends React.Component<PlayProperties, any> {
         if (w !== this.seekgraph.width || h !== this.seekgraph.height) {
             this.seekgraph.resize(w, h);
         }
+        if (w === 0 || h === 0) { // Wait for positive size
```

Comment:
@anoek Is this condition strict enough? I could change it to `if (Math.min(w, h) <= 0)`, similar to [your check](https://github.com/online-go/online-go.com/blob/067eadacc12844d60733908a49e3eb5e3deceabc/src/views/Game/Game.tsx#L387).